features/waarden.feature

```diff
@@ -96,20 +160,46 @@ Feature: leveren van waarden bij een WOZ-object
     ]
     """
 
-  Scenario: Belanghebbende krijgt verlaging van de WOZ-waarde na bezwaar: bij beschikkingen met zelfde waardepeildatum en zelfde ingangsdatum gebruiken we de meest recente beschikking
-    Gegeven WOZ-object met objectnummer "082600014669" heeft de belanghebbende eigenaar bezwaar ingediend over de WOZ-waarde op 01-01-2017
-    En dat bezwaar is afgehandeld met de status "bezwaar afgehandeld, vastgestelde waarde veranderd"
-    En de belanghebbende eigenaar heeft bezwaar ingediend over de WOZ-waarde op 01-01-2018
-    En dat bezwaar heeft de status "bezwaar ingediend"
+  Scenario: Belanghebbende krijgt verlaging van de WOZ-waarde na bezwaar
+    Gegeven WOZ-object met objectnummer "082600014669" heeft de belanghebbende eigenaar bezwaar ingediend over de WOZ-waarde met peildatum 01-01-2017
+    En dat bezwaar is afgehandeld met de status "bezwaar afgehandeld, vastgestelde waarde veranderd" (12)
+    En de belanghebbende eigenaar heeft bezwaar ingediend over de WOZ-waarde met peildatum 01-01-2018
+    En dat bezwaar heeft de status "bezwaaringediend" (20)
     En WOZ-object met objectnummer "082600014669" heeft de volgende beschikkingen:
-      | Peildatum  | WOZ-waarde   | Belanghebbende | Ingangsdatum | Datum beschikking |
-      | 01-01-2019 | 195.000 euro | R. Geels       | 01-01-2020   | 13-01-2020        |
-      | 01-01-2017 | 152.000 euro | R. Geels       | 01-01-2018   | 03-02-2019        |
-      | 01-01-2018 | 176.000 euro | R. Geels       | 01-01-2019   | 14-01-2019        |
-      | 01-01-2017 | 171.000 euro | R. Geels       | 01-01-2018   | 15-01-2018        |
-      | 01-01-2016 | 164.000 euro | R. Geels       | 01-01-2017   | 16-01-2017        |
+    | Peildatum  | WOZ-waarde   | Ingangsdatum | Datum beschikking |
+    | 01-01-2019 | 195.000 euro | 01-01-2020   | 13-01-2020        |
+    | 01-01-2017 | 152.000 euro | 01-01-2018   | 03-02-2019        |
+    | 01-01-2018 | 176.000 euro | 01-01-2019   | 14-01-2019        |
+    | 01-01-2017 | 171.000 euro | 01-01-2018   | 15-01-2018        |
```

Comment:
Deze waarde komt niet terug in de geleverde waarden. Klopt dit?

Reply:
de API levert alleen de actuele waarde. dat is de laatst bepaalde waarde op peildatum en ingangsdatum.
@oversticht klopt het als er verschillende belanghebbenden een beschikking krijgen over dezelfde peildatum en ingangsdatum, maar met verschillende waarde, dat dan alleen de de meest recente beschikking van de twee wordt geleverd?